composer.json

```diff
@@ -163,6 +163,9 @@
         "test": [
             "tools/phpunit/vendor/bin/phpunit"
         ],
+        "test:benchmark": [
```

Comment:
It would be nice to create another GitHub Actions workflow for this

Reply:
Added.